R/model_run.R

```diff
@@ -1,4 +1,37 @@
 model_run <- function(parameters, model_version) {
-  # Returning sample response for now
-  from_file <- read_local_json("sample_scenario_results_response.json")
+  country <- parameters$country
+  pathogen <- parameters$pathogen
+  response <- parameters$response
+  # TODO: include vaccine in params to daedalus
+  vaccine <- parameters$vaccine
+  # TODO: include hospital capacity in params to daedalus
+  hospital_capacity <- parameters$hospital_capacity
```

Comment:
Up to you! I think the api is the place for the translation from web app terminology to package terminology to take place, so it's probably ok for there to be some differences. But I guess in general it's good to keep them harmonised where possible, especially if we're imagining a bit of a pipeline of web app users becoming package users.

Reply:
I'm confused how these two terms could refer to the same thing - a capacity seems different from a threshold.

Reply:
I see the documentation of the parameter here: https://github.com/jameel-institute/daedalus/pull/27/files#diff-26b50e25422503756a8c78207bc54269300bdedbb5c122249cd8d0e9e1b7c8f3

It seems that response_threshold uses (spare) hospital capacity as a fallback value, so I'd say keeping the name "response threshold" is more accurate, since a government might (probably would) choose to set their threshold differently from the spare hospital capacity.